custom_components/healthchecksio/binary_sensor.py

```
@@ -1,12 +1,26 @@
 """Binary sensor platform for Healthchecksio."""
 
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
```

Ruff (0.11.9)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)